client/tests/setup/mocks.ts

```diff
@@ -0,0 +1,21 @@
+// To return a different value for each test, you can import this file and use mockReturnValue
+// to set the return value for the function you want to mock using mockReturnValue
+// or mockReturnValueOnce. Example:
```

Comment:
I don't understand this part--the example doesn't show how to do different values for each test/resets